src/registry_schemas/schemas/party.json

```diff
@@ -12,7 +12,8 @@
                     "title": "The type of party",
                     "enum": [
                         "organization",
-                        "person"
+                        "person",
+                        "businessOrCorporation"
```

Comment:
Is this new type needed? Can't organization be used?

Reply:
removed businessOrCorporation